From 95beaa16620935ff63127c1e2c3234f7594813c7 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Thu, 31 Oct 2024 10:34:59 +0200
Subject: [PATCH] [SW-207059] gaudi3: pci: update eq preset mask

For better GEN5 link stability in GT servers the equalization preset
mask is updated.

Change-Id: I4403829aaa956b180e7104f7d67529208fcc37d4
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index baef104..622185f 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -1017,7 +1017,7 @@
 static void pif_arc_config_dbi_params(void)
 {
 	regmap_writel(0x02402801, D0_PCIE_DBI_SIG_REG(mmPCIE_DBI_GEN3_RELATED_OFF));
-	regmap_writel(0x04020031, D0_PCIE_DBI_SIG_REG(mmPCIE_DBI_GEN3_EQ_CONTROL_OFF));
+	regmap_writel(0x05026031, D0_PCIE_DBI_SIG_REG(mmPCIE_DBI_GEN3_EQ_CONTROL_OFF));
 	regmap_rmwritel(D0_PCIE_DBI_SIG_REG(mmPCIE_DBI_DEVICE_CAPABILITIES2_REG),
 					ENABLE_PWRBRK_CTRL_VAL, EMR_PWR_M);
 
